### PR TITLE
add exempt indicator

### PR DESCRIPTION
**Pull request by tinglisD2L**
https://rally1.rallydev.com/#/15545269080/detail/userstory/221603123812

(no tooltip yet because I haven't gotten the details about how to do that)

### CODE REVIEW

File: components/d2l-completion-status.html [hunk 1]
```diff
@@ -31,17 +31,22 @@
...
-			<div class$="[[completionStatus]]">
-				[[localize('optional')]]
-			</div>
+		<template is="dom-if" if="[[isExempt]]">
+			<d2l-icon class$="[[completionStatus]]" icon="d2l-tier1:exemption-add"></d2l-icon>
 		</template>
-		<template is="dom-if" if="[[!isOptional]]">
-			<template is="dom-if" if="[[isComplete]]">
-				<d2l-icon class$="[[completionStatus]]" icon="d2l-tier1:check"></d2l-icon>
+		<template is="dom-if" if="[[!isExempt]]">
```
Comment (RyanSweetD2L) [inline components/d2l-completion-status.html]: I would prefer chaining our logic to having multiple template sections.  Something like ```if="[[!isExempt && isOptional]]"``` could work.

File: components/d2l-completion-status.html [hunk 2]
```diff
@@ -31,17 +31,22 @@
...
+			<template is="dom-if" if="[[isOptional]]">
+				<div class$="[[completionStatus]]">
+					[[localize('optional')]]
+				</div>
 			</template>
-			<template is="dom-if" if="[[!isComplete]]">
-				<d2l-icon class$="[[completionStatus]]" icon="d2l-tier1:dot"></d2l-icon>
+			<template is="dom-if" if="[[!isOptional]]">
```
Comment (RyanSweetD2L) [inline components/d2l-completion-status.html]: same here?  ```if="[[!isOptional && isComplete]]"```

File: components/d2l-completion-status.html (hunk 1 above)
Comment (tinglisD2L) [reply components/d2l-completion-status.html]: that doesn't work unfortunately

Comment (RyanSweetD2L) [reply components/d2l-completion-status.html]: Hmm... you're right.  We could refactor to a function, but I wonder if it is worth it.  I just don't like the nested templates....

**Pull request merged**